<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:56,331] Register - Step 01: Navigate to Register page
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:57,345] Register - Step 02: Enter to Firstname textbox with value is Automation
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:57,459] Register - Step 03: Enter to Lastname textbox with value is FC
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:57,563] Register - Step 04: Enter to Email Adrress textbox with value is FC
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:57,660] Register - Step 05: Enter to Password textbox with value is 123456
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:57,770] Register - Step 06: Enter to ConfirmPassword textbox with value is 123456
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:57,877] Register - Step 07: Click to Register button
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:58,606] Register - Step 08: Verify Register Success Message is displayed
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:58,656]  -------------------------- FAILED -------------------------- 
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:58,659] Login - Step 01: Navigate to Login page
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:59,745] Login - Step 02: Enter to email textbox with value is afc3893@gmail.com
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:59,861] Login - Step 03: Enter to password textbox with value is 123456
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:33:59,972] Login - Step 04: Click to login button
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:34:00,574] Login - Step 05: Verify My Account Link is displayed
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:34:00,640]  -------------------------- FAILED -------------------------- 
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:34:00,641] Login - Step 06: Navigate to My Account Page
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:34:01,104] Login - Step 07: Verify Customer Info page is displayed
<br>]]>
    </line>
    <line>
      <![CDATA[[INFO ] [2022-11-03 22:34:01,166]  -------------------------- FAILED -------------------------- 
<br>]]>
    </line>
  </reporter-output>
  <suite started-at="2022-11-03T15:33:53Z" name="NopCommerce - User" finished-at="2022-11-03T15:34:01Z" duration-ms="7947">
    <groups>
    </groups>
    <test started-at="2022-11-03T15:33:53Z" name="Run on Chrome" finished-at="2022-11-03T15:34:01Z" duration-ms="7947">
      <class name="com.nopcommerce.user.Level_15_ReportNG_Screenshot">
        <test-method is-config="true" signature="beforeClass(java.lang.String)[pri:0, instance:com.nopcommerce.user.Level_15_ReportNG_Screenshot@7bd7d6d6]" started-at="2022-11-03T15:33:53Z" name="beforeClass" finished-at="2022-11-03T15:33:56Z" duration-ms="2503" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="User_01_Register()[pri:0, instance:com.nopcommerce.user.Level_15_ReportNG_Screenshot@7bd7d6d6]" started-at="2022-11-03T15:33:56Z" name="User_01_Register" finished-at="2022-11-03T15:33:58Z" duration-ms="2328" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Your registration completed...] but found [Your registration completed]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Your registration completed...] but found [Your registration completed]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at commons.BaseTest.verifyEquals(BaseTest.java:176)
at com.nopcommerce.user.Level_15_ReportNG_Screenshot.User_01_Register(Level_15_ReportNG_Screenshot.java:55)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:56,331] Register - Step 01: Navigate to Register page
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:57,345] Register - Step 02: Enter to Firstname textbox with value is Automation
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:57,459] Register - Step 03: Enter to Lastname textbox with value is FC
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:57,563] Register - Step 04: Enter to Email Adrress textbox with value is FC
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:57,660] Register - Step 05: Enter to Password textbox with value is 123456
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:57,770] Register - Step 06: Enter to ConfirmPassword textbox with value is 123456
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:57,877] Register - Step 07: Click to Register button
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:58,606] Register - Step 08: Verify Register Success Message is displayed
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:58,656]  -------------------------- FAILED -------------------------- 
<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- User_01_Register -->
        <test-method signature="User_02_Login()[pri:0, instance:com.nopcommerce.user.Level_15_ReportNG_Screenshot@7bd7d6d6]" started-at="2022-11-03T15:33:58Z" name="User_02_Login" finished-at="2022-11-03T15:34:01Z" duration-ms="2508" status="FAIL">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Multiple failures (2):
Failure 1 of 2
java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at commons.BaseTest.verifyFalse(BaseTest.java:162)
at com.nopcommerce.user.Level_15_ReportNG_Screenshot.User_02_Login(Level_15_ReportNG_Screenshot.java:74)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

Failure 2 of 2
java.lang.AssertionError: expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Multiple failures (2):
Failure 1 of 2
java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at commons.BaseTest.verifyFalse(BaseTest.java:162)
at com.nopcommerce.user.Level_15_ReportNG_Screenshot.User_02_Login(Level_15_ReportNG_Screenshot.java:74)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

Failure 2 of 2
java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at commons.BaseTest.verifyFalse(BaseTest.java:162)
at com.nopcommerce.user.Level_15_ReportNG_Screenshot.User_02_Login(Level_15_ReportNG_Screenshot.java:80)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:58,659] Login - Step 01: Navigate to Login page
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:59,745] Login - Step 02: Enter to email textbox with value is afc3893@gmail.com
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:59,861] Login - Step 03: Enter to password textbox with value is 123456
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:33:59,972] Login - Step 04: Click to login button
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:34:00,574] Login - Step 05: Verify My Account Link is displayed
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:34:00,640]  -------------------------- FAILED -------------------------- 
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:34:00,641] Login - Step 06: Navigate to My Account Page
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:34:01,104] Login - Step 07: Verify Customer Info page is displayed
<br>]]>
            </line>
            <line>
              <![CDATA[[INFO ] [2022-11-03 22:34:01,166]  -------------------------- FAILED -------------------------- 
<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- User_02_Login -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:com.nopcommerce.user.Level_15_ReportNG_Screenshot@7bd7d6d6]" started-at="2022-11-03T15:34:01Z" name="afterClass" finished-at="2022-11-03T15:34:01Z" duration-ms="598" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.nopcommerce.user.Level_15_ReportNG_Screenshot -->
    </test> <!-- Run on Chrome -->
  </suite> <!-- NopCommerce - User -->
</testng-results>
